projects/kit/components/arrow/arrow.style.less

```diff
@@ -1,11 +1,14 @@
 @import 'taiga-ui-local';
 
 .icon {
-    .transition(transform);
     cursor: pointer;
     pointer-events: none;
 
-    &_rotated {
+    > * {
```

Comment:
not sure, why changed?

Reply:
The icon became one level deeper and this style nesting helps you replace the icon without providing additional classes. The line diff might be odd in this case, please check this PR in split diff.

Reply:
I think it's better to apply display: block to the span. Transform doesn't work on inline elements.

Reply:
@waterplea, I'm ashamed I didn't notice that it's inline. Thanks for the heads up.